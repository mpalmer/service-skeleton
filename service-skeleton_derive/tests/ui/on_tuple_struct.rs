#![allow(unused_imports, unused_crate_dependencies)]
use service_skeleton_derive::ServiceConfig;

#[derive(ServiceConfig)]
struct Config(String, String);

fn main() {}
