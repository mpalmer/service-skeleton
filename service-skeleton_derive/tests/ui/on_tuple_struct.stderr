error: Unsupported shape `unnamed fields`. Expected named fields.
 --> tests/ui/on_tuple_struct.rs:4:10
  |
4 | #[derive(ServiceConfig)]
  |          ^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `ServiceConfig` (in Nightly builds, run with -Z macro-backtrace for more info)
